device/mellanox/x86_64-nvidia_sn5400-r0/ACS-SN5400/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,39 @@
+{#
```

Comment:
We can just make this file a symbol link to `x86_64-nvidia_sn5600-r0/ACS-SN5600/buffers_defaults_t0.j2`